packages/cacti-plugin-copm-corda/src/main/kotlin/org/hyperledger/cacti/plugin/copm/corda/CordaAssetClaim.kt

```diff
@@ -0,0 +1,26 @@
+package org.hyperledger.cacti.plugin.copm.corda
+import org.hyperledger.cacti.plugin.copm.validators.ValidatedClaimPledgedAssetV1Request
+import net.corda.core.contracts.CommandData
+import org.hyperledger.cacti.weaver.imodule.corda.states.AssetClaimParameters
+
+class CordaAssetClaim(private var data: ValidatedClaimPledgedAssetV1Request,
```

Comment:
@jenniferlianne This looks like it might also be generated code but I'm not sure. If it is, please move it under a directory path that contains `generated` as a path element so that it's easier to distinguish. There might be other pockets of generated code that I didn't find but please apply this change to all other generated code as well.

Reply:
It's actually not generated.